JetBrains IDE plugin / Move Profiler docs to separate page (#29758)
DataDog/documentation: content/en/developers/ide_plugins/idea/_index.md

You can dismiss individual insights and set filters to view the categories of insights that you are interested in.

## Live Debugger

> The [**Live Debugger**][20] enables you to add logpoints—auto-expiring, "non-breaking breakpoints"—to your runtime code to collect information for debugging.

Reviewer: ```suggestion
The [**Live Debugger**][20] enables you to add logpoints—auto-expiring, non-breaking breakpoints—to your runtime code to collect information for debugging.
```

{{< img src="/developers/ide_plugins/idea/live_debugger/tool-window-tab.png" alt="The Live Debugger tab" style="width:100%;" >}}

Find out more in the [Live Debugger documentation][20].

## Continuous Profiler

> The [**Continuous Profiler**][22] highlights resource consumption (CPU, memory allocation and thrown exceptions, for example) using profiling data collected from deployed services.  This information helps developers write more efficient code and eliminate bottlenecks.

Reviewer: ```suggestion
The [**Continuous Profiler**][22] highlights resource consumption (such as CPU, memory allocation, and thrown exceptions) using profiling data collected from deployed services.  This information helps developers write more efficient code and eliminate bottlenecks.
```
